test/index.spec.ts

```diff
@@ -70,13 +70,13 @@ describe('logger', () => {
     expect(debug.formatters).to.have.property('p').that.is.a('function')
     expect(debug.formatters).to.have.property('c').that.is.a('function')
     expect(debug.formatters).to.have.property('k').that.is.a('function')
-    expect(debug.formatters).to.have.property('ma').that.is.a('function')
+    expect(debug.formatters).to.have.property('a').that.is.a('function')
   })
 
   it('test ma formatter', () => {
     const ma = multiaddr('/ip4/127.0.0.1/tcp/4001')
 
-    expect(debug.formatters.ma(ma)).to.equal(ma.toString())
+    expect(debug.formatters.a(ma)).to.equal(ma.toString())
```

Comment:
Can we add a test that would have failed?

Reply:
Good idea